src/status_im/constants.cljs

```diff
@@ -88,9 +89,15 @@
           mainnet-networks)))
 
 (def default-wnodes
-  {:testnet {"main"   {:id      "main"
-                       :name    "Status testnet mailserver A"
-                       :address "enode://88329092a4328bec2bf5422063a87e50637992d3d7e2e2937371ad6db36e268c4fe7ce4e1f45c71e462339d051505a65b9ac13b86c5788d5bee7f481c01ba58b@159.65.125.20:30504"}}
+  {:testnet {"main" {:id      "main"
+                     :name    "Status testnet mailserver A"
+                     :address "enode://88329092a4328bec2bf5422063a87e50637992d3d7e2e2937371ad6db36e268c4fe7ce4e1f45c71e462339d051505a65b9ac13b86c5788d5bee7f481c01ba58b@159.65.125.20:30504"}
+             "paid" {:id      "paid"
```

Comment:
Could we hide this behind a feature flag? For an example of a feature flag, see https://github.com/status-im/status-react/pull/4054/files

Reply:
Actually, considering it is just a configuration option you see when you are on testnet, maybe we don't need to hide this. I see the toggling logic is already dealt with in other parts of the code, so we might be fine without a flag.

Reply:
If we want to we can always add this flag back under https://github.com/status-im/status-react/issues/4003 later